libass/ass_bitmap.c

```diff
@@ -370,13 +370,15 @@ void be_blur_post(uint8_t *buf, intptr_t w, intptr_t h, intptr_t stride)
  */
 int be_padding(int be)
 {
+    if (be <= 0)
+        return 0;
     if (be <= 3)
-        return be;
+        return be + 1;
     if (be <= 7)
-        return 4;
-    if (be <= 123)
         return 5;
-    return FFMAX(128 - be, 0);
+    if (be <= 123)
+        return 6;
+    return FFMAX(128 - be, 0) + 2;
```

Comment:
Hm. This is odd.

For one thing, you’re adding 2 in the last line but only 1 elsewhere. Also, the comment above this function should be updated to say something like, “One extra pixel is added to allow for vectorized code.”

Is the problem that the asm code is vectorized and writes to one byte past the last column/row, overrunning the buffer? But doesn’t this function affect the computed width and height of the bitmap, and therefore doesn’t this change merely cause the asm code to write _another_ byte past the last column/row, _still_ overrunning the buffer?

Reply:
It doesn't overrun, it just produces incorrect output when the input has nonzero data within 2px of the edge (don't ask me why, that comes straight from VSFilter).

Reply:
Removed this commit.